### PR TITLE
completed es6-modules exercise.

### PR DESCRIPTION
**Pull request by joseph-w-valdez**
image

### CODE REVIEW

File: es6-modules/src/components/todo-app.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      createElement('h1', { class: 'mb-4' }, 'JS Todo'),
+      this.todoForm.render(),
+      this.todoList.render(this.todos)
+    );
+    const { nextId, todos } = this;
+    this.onUpdate({ nextId, todos });
+  }
+}
+
+export default TodoApp;
```
Comment (RobertGardner) [inline es6-modules/src/components/todo-app.js:55]: You can put this here, but it's easier to just put `export default` on the class. Oh, you did that in the next one. Nice.

File: es6-modules/src/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import TodoApp from './components/todo-app';
```
Comment (RobertGardner) [inline es6-modules/src/index.js:1]: Usually you should have a blank line after all your imports.

File: es6-modules/src/lib/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import noop from './noop';
+import toArray from './to-array';
+import createElement from './create-element';
+
+export {
+  noop,
+  toArray,
+  createElement
+};
```
Comment (RobertGardner) [inline es6-modules/src/lib/index.js:9]: There's a syntax we haven't learned yet that is convenient here:

`export noop from './noop';`

or even easier if you just want to export everything (which is usually what you want):

`export * from './noop';`

**Review by RobertGardner — APPROVED** [review]

File: es6-modules/src/components/todo-app.js (hunk 1 above)
Comment (joseph-w-valdez) [reply es6-modules/src/components/todo-app.js:55]: Ah yeah, I was testing out different ways it works. I'll adjust it to the most correct way now.

**Pull request merged**